client/src/Movies/Movie.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
Y'all can condense line 2 and 3 `import { Link, Route } from 'react-router-dom';` but `Route` isn't even used